<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IVEditorDialog</class>
 <widget class="QDialog" name="IVEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0" colspan="2">
       <widget class="QComboBox" name="comboBox_selectiv">
        <property name="minimumSize">
         <size>
          <width>119</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
    background-color: white;
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
}
</string>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAlphabetically</enum>
        </property>
        <property name="placeholderText">
         <string>Select Independent Variable...</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QPushButton" name="btn_removevalue">
        <property name="text">
         <string>Remove selected</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QTableWidget" name="tableWidget_storedvalues">
        <property name="styleSheet">
         <string notr="true">QAbstractItemView::indicator{
	width: 20px;
	height: 20px;
}

QAbstractItemView::indicator:checked {
    /* Adjust the appearance of the checked indicator */
    image: url(:/images/images/checked_icon.png) /* Specify a custom image if desired */
}

QAbstractItemView::indicator:unchecked {
    /* Adjust the appearance of the unchecked indicator */
    image: url(:/images/images/unchecked_icon.png); /* Specify a custom image if desired */
}


QTableWidget {
    font-size: 12pt; /* Set the font size */
    font-weight: bold; /* Set the font weight */
}

QTableWidget::item:selected {
    background-color: rgba(50, 50, 225, 0.1); /* Change the background color and opacity */
	color:black;
}

QTableWidget::item:focus {
    selection-color: blue; /* Hide the dotted selection border */
}

QAbstractItemView::indicator:hover {
    /* Adjust the appearance of the hover indicator */
    background-color: lightgray; /* Example: change background color when hovering */
}

QAbstractItemView::indicator:checked:hover {
    /* Adjust the appearance of the checked checkbox when hovering */
    background-color: rgb(255, 166, 167); /* Example: change background color of checked checkbox when hovering */

}


QAbstractItemView::indicator:unchecked:hover {
    /* Adjust the appearance of the unchecked checkbox when hovering */
    background-color: lightgreen; /* Example: change background color of unchecked checkbox when hovering */
}

</string>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>220</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Delete</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>IV-Value</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="6" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_nameinput">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Select IV above...</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_valueinput">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>IV-Value (&quot;Young&quot;, &quot;Old&quot;, ...)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="btn_editconfirm">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Confirm changes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: lightgrey;
    border: 1px solid white;
	color: white;
}

QPushButton:hover {
    background-color: lightgrey;
    color: dark grey; 
	text-decoration: underline;
}

QPushButton:pressed {
    background-color: white;
}

QPushButton:focus {
    border: 1px solid white;
}

QPushButton::menu-indicator {
    image: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/images/checked_icon.png</normaloff>:/images/images/checked_icon.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_ivname">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Change name (confirm with button on the right):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btn_delete_iv">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: lightgrey;
    border: 1px solid white;
	color: white;
}

QPushButton:hover {
    background-color: lightgrey;
    color: dark grey; 
	text-decoration: underline;
}

QPushButton:pressed {
    background-color: white;
}

QPushButton:focus {
    border: 1px solid white;
}

QPushButton::menu-indicator {
    image: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/images/trash_icon.png</normaloff>:/images/images/trash_icon.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_header">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Independent Variable - Editor</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="btn_addvalue">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add input value to current IV&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: lightgrey;
    border: 1px solid white;
	color: white;
}

QPushButton:hover {
    background-color: lightgrey;
    color: dark grey; 
	text-decoration: underline;
}

QPushButton:pressed {
    background-color: white;
}

QPushButton:focus {
    border: 1px solid white;
}

QPushButton::menu-indicator {
    image: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/images/Plus_Icon.svg</normaloff>:/images/images/Plus_Icon.svg</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0" alignment="Qt::AlignTop">
       <widget class="QLabel" name="label_storedvals">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Current values for selected IV:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_valuename">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Add value:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_ivselect">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Select Independent Variable to edit:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_valueinputwarning">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{color:red;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_variableinputwarning">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{color:red;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IVEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IVEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
